doc/man3/EVP_MAC.pod

```diff
@@ -308,6 +308,10 @@ EVP_MAC_do_all_ex() returns nothing at all.
           params[params_n++] =
               OSSL_PARAM_construct_utf8_string("digest", digest,
                                                strlen(digest) + 1, NULL);
+      /*
+       * Note that in general, the key would be binary data and that
```

Comment:
can't we just fix the example, even to put a variable named key_size or something?